Makefile

```diff
@@ -422,13 +422,16 @@ $(eval $(call RUN_IN_DEV_ENV_TEMPLATE,tidy,clean))
 define generate-ctrl-make
 	$(GO) generate ./pkg/ctrlserver/graph
 endef
-$(eval $(call RUN_IN_DEV_ENV_TEMPLATE,generate-ctrl))
+$(eval $(call RUN_IN_DEV_ENV_TEMPLATE,generate-ctrl,generate-deepcopy))
 
-define generate-make
+define generate-deepcopy-make
 	cd ./api/v1alpha1 ;\
 		controller-gen object:headerFile=../../hack/boilerplate/boilerplate.generatego.txt paths="./..." ;
 endef
-$(eval $(call RUN_IN_DEV_ENV_TEMPLATE,generate,chaos-build generate-ctrl swagger_spec))
+$(eval $(call RUN_IN_DEV_ENV_TEMPLATE,generate-deepcopy,chaos-build))
+
+generate: generate-ctrl swagger_spec generate-deepcopy chaos-build
```

Comment:
Should we need to add `generate-deepcopy` as the dependency of `generate-ctrl`?

Does `generate-ctrl` depends on the `DeepCopy()` methods ? @Hexilee

Reply:
Actually, I never expect the `Makefile` under Chaos Mesh can run with `-j xx`, so the dependency is always built in order 😢 .

Reply:
Oops actually I also never think about `-j`, I just notice the order in `generate-ctrl swagger_spec generate-deepcopy chaos-build`.

It seems `schema.resolvers.go` does not use any methods in `zz_generated.deepcopy.go`, `zz_generated.chaosmesh.go` and `zz_generated.<other>.chaosmesh.go`, I think `generate-ctrl` does not depend on `generate-deepcopy` and `chaos-builder`